docs/content/tutorials/nginx-ingress-istio.md

```diff
@@ -0,0 +1,273 @@
+---
+title: NGINX Ingress Controller and Istio Service Mesh
+description: |
+  Use NGINX Ingress Controller with Istio Service Mesh.
+weight: 1800
+doctypes: ["concept"]
+toc: true
+docs: "DOCS-889"
+---
+
+## Using NGINX Ingress Controller with Istio service mesh    
+
+NGINX Ingress Controller can now be used as the ingress controller for applications that are running inside an Istio service mesh. This allows to continue using the advanced capabilities that NGINX IC provides on Istio-based environments without resorting to any workarounds.
+This is accomplished using the special setting [use-cluster-ip](https://docs.nginx.com/nginx-ingress-controller/configuration/virtualserver-and-virtualserverroute-resources/#upstream) for the backend.
+
+
+Here is a standard deployment of NGINX Ingress controller without a sidecar proxy injected into the pod.    
+
+{{< img src="./img/nginx_plain.png" alt="NGINX stand alone." >}}
+
+## Install Istio
+
+Link to Istio install guide:    
+[Installing istio](https://istio.io/latest/docs/setup/install/)    
+
+It is very important to make sure you install Istio **BEFORE**, you install NGINX Ingress Controller. This is to ensure that the istio sidecar is properly injected into the NGINX Ingress controller pod.
+
+You can then install Istio by your preferred method (helm, operator etc.). Deploy Istio into your cluster. In this case, I ran the following command to install Istio into my cluster:
+
+```
+istioctl install --set profile=minimal
+```
+
+We need to make sure that Istio injects sidecar proxies into our namespace for our testing. To do so, we need to tell Istio what namespaces to inject sidecars into. We can do that with the following command:
+```
+kubectl label ns <namespace_specified> istio-injection=enabled
+```

```

Comment:
Before proceeding, you should deploy NGINX Ingress Controller.
If you have not done so already go <here> and follow steps 1 - x to get NGINX Ingress Controller installed and running.

Now that Istio is installed and prepared, and NGINX Ingress controller is deployed - the next step is to enable Istio sidecar injection for the NIC pods, to capture traffic egressing from NIC to the backend service(s).